EMRM.py

```diff
@@ -151,12 +182,28 @@ def get_delta_x(self, word, lpos):
         return(delta_x)
         
 class OneFixation:
+    """ Class of a fixation dwelling at a location (in terms of characters) for a period of time.
```

Comment:
"location (in terms of characters)" -> "character position"?

Reply:
modified